index.html

```diff
@@ -136,7 +136,7 @@ <h2 class="card__content--heading">Tonic</h2>
             <li class="tech__btn">css</li>
```

Comment:
- [ ] As per the project requirements, created code about Projects in the HTML should be removed and added to JavaScript. I see that you already have it in JS, kindly remove it from the `index.html`
`The projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for projects list should be removed, and instead it should be added via JavaScript. `